precompile/params.go

```diff
@@ -9,6 +9,10 @@ import "github.com/ethereum/go-ethereum/common"
 const (
 	ModifyAllowListGasCost = 20_000
 	ReadAllowListGasCost   = 5_000
+
+	MintGasCost = 20_000
+
+	CallNewAccountGas = 25_000 // Paid for CALL when the destination address didn't exist prior.
```

Comment:
Can we use the parameter in the `params` package instead of copying it in here?

Reply:
cyclic import :/

Reply:
I wonder why chain configs and params are in the same package...

Reply:
removed this completely